doc/TROUBLESHOOT.md

```diff
@@ -101,6 +101,35 @@ Otherwise, the reference to the first non-needed object on the path (`staticX`,
 staticX -> A -> B -> disposedD
 ```
 
+Be careful not to fall into pattern to limit your fix to just releasing the reference to the disposed object. 
```

Comment:
Consider telling the user what they should do instead of what they should not do.

e.g. `The leak tracker does not detect all leaks, only the leaks of objects it explicitly is tracking. When fixing detected leaks, make sure to look for other objects that may have been leaked without being visible to the tracker.`

An code example with a `BAD` and `GOOD` example would probably help here too.